From 031b5e5e999d3fd125199e908b15ab5e4007cd02 Mon Sep 17 00:00:00 2001
From: Francesco Poli (t1000) <frx@firenze.linux.it>
Date: Sat, 22 May 2010 22:21:12 +0200
Subject: [PATCH 4/8] drop one reassigned bug from TODO

This bug has been reassigned to another package and should be fixed
there...
---
 debian/TODO |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/debian/TODO b/debian/TODO
index 19fdf13..30d8345 100644
--- a/debian/TODO
+++ b/debian/TODO
@@ -2,7 +2,6 @@ Release goals for version 0.1.5
 -------------------------------
 * check the pbuilder tests and make them work
 * close #532678 - apt-listbugs: faster method to select bugs
-* close #539487 - apt-listbugs: does not work via proxy using windows style user names
 * close #536613 - apt-listbugs: lists no bugs and returns 0 for non existing packages [waiting on debbugs, #553661]
 * close #553694 - apt-listbugs: remove html from translations
 
-- 
1.7.1

